2025/ideas-list.md

```diff
@@ -15,6 +15,7 @@ page of each organization under the NumFOCUS umbrella at this page.
 - [PySAL](https://github.com/pysal/pysal/wiki/Google-Summer-of-Code-2025)
 - [Qutip](https://github.com/qutip/qutip/wiki//Google-Summer-of-Code-2025)
 - [SciML](https://sciml.ai/dev/#google_summer_of_code)
+- [Stan](https://github.com/stan-dev/stan/wiki/GSOC-2025-Proposed-Projects)
```

Comment:
Google expects some level of organization for the project. Please take a look at the ideas submitted by your peers to refine your proposal accordingly. Thank you!

- [aeon](https://github.com/aeon-toolkit/aeon-admin/blob/main/gsoc/gsoc-2025-projects.md)
- [AiiDA](https://github.com/aiidateam/aiida-core/wiki/GSoC-2025-Projects)